### DISCUSSION TITLE
Welcome to fuse-box Discussions!

### DISCUSSION THREAD

**Discussion by nchanged** [fuse-box/fuse-box#1968] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SephReed** [comment]
Hello, I'm Seph.  I haven't contributed much since a bug fix and some doc updates a while ago.  But I still like to keep updated and be here to root the project on.
